src/StaticFAQ/FAQArticle.js

```diff
@@ -26,34 +28,62 @@ const style = css`
   }
 `;
 
-const FAQArticle = (props: Props) => (
-  <Link
-    data-cy="faq-article-link"
-    to={
-      props.isSearchResult
-        ? `/faq/search/article/${props.article.id}`
-        : `/faq/${props.categoryId || ''}/article/${props.article.id}`
-    }
-    style={{ textDecoration: 'none', display: 'block' }}
-  >
-    <Card>
-      <div>
-        <Heading weight="medium" size="small">
-          {props.article.title}
-        </Heading>
-      </div>
-      <div className="perex">
-        <Text type="secondary" size="small" element="span">
-          {props.article.perex}
-        </Text>
-      </div>
-    </Card>
-    <style jsx>{style}</style>
-  </Link>
+const articleClicked = (
+  resetQueriesCount: () => void,
+  queriesBeforeClick: number,
+  searchText: string,
+) => () => {
+  if (searchText) {
+    simpleTracker('smartFAQBookingOverview', {
+      action: 'articleClicked',
+      queriesBeforeClick,
+    });
+    resetQueriesCount();
+  }
+};
+const RawFAQArticle = (props: Props) => (
```

Comment:
I would split it into two components, to keep logic separated from markup

something like:
```
class TrackedFAQLink extends React.Component {
  onClick() {
    ...
  }

  render() {
   return (<FAQArticle  onClick={this.onClick} />)
  }
}
```